### PR TITLE
chore(Node): upgrade to node 12 TIDE-1135

### PR DESCRIPTION
**Pull request by rgdSolutions**
Updated react-ui to use node 12 and node-sass 4.12, also created .node-version and .nvmrc files.

[CARD](https://rentpath.atlassian.net/browse/TIDE-1135)

### CODE REVIEW

**Comment by lfender6445** [comment]
We also need to update travis config https://github.com/rentpath/react-ui/blob/master/.travis.yml#L3

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (lfender6445) [inline .yarnrc]: more on this here if you're interested https://yarnpkg.com/en/docs/cli/policies

Comment (rgdSolutions) [reply .yarnrc]: Thanks! That was the exact link I used to find out about --> yarn policies set-version

Comment (lfender6445) [reply .yarnrc]: oh i know, was leaving this for people who might wonder how the file originated

**Pull request merged**